integration/spark2/src/main/commonTo2.2And2.3/org/apache/spark/sql/hive/CarbonInMemorySessionState.scala

```diff
@@ -157,7 +158,13 @@ class InMemorySessionCatalog(
   }
 
   override def lookupRelation(name: TableIdentifier): LogicalPlan = {
-    val rtnRelation = super.lookupRelation(name)
+    val rtnRelation: LogicalPlan =
+      try {
```

Comment:
why we need this try/catch, its super class doesn't throw any exception right, moreover we are throwing out the same exception object, can you please define the intention of this statement?

Reply:
i have described in description section, may be that will explain, can you please have a look?